addons/website/views/website_templates.xml

```diff
@@ -1780,7 +1780,7 @@
                 <div class="modal-dialog d-flex s_popup_size_full">
                     <div class="modal-content oe_structure">
                         <section class="o_colored_level o_cc o_cc1">
-                            <div class="container">
+                            <div class="container pb-3 pb-lg-0">
```

Comment:
Good catch. Thanks for the complete explanation and details in your WIP commit! 

I took a quick look at it and it looks good to me except that the cookie bar is shifted to the left and bottom (screen attached. It may be hard to see but to clarify: the black part is the footer of the site and the grey part is the background of my browser)
![cookiebarcommit](https://user-images.githubusercontent.com/91664927/168629126-e767174b-3e71-47d5-b46d-64c112577d7f.png)

Couldn't removing the margins only for the cookiebar as mentioned in my comment (https://github.com/odoo/odoo/pull/88905#discussion_r873866433)  be complementary to your fix?